apps/frontend/src/app/3_organisms/LOCChart/LOCChart.tsx

```diff
@@ -165,12 +150,12 @@ export const LOCChart: FC<LOCChartProps> = ({ isDefaultView = false }) => {
             debtAmount,
             price,
           ),
-        };
+        } as TroveData;
```

Comment:
```suggestion
        return {
          id: trove.id,
          collateral: collateralAmount,
          debt: debtAmount,
          collateralRatioSortKey_legacy: trove.collateralRatioSortKey_legacy ?? '',
          collateralRatio: calculateCollateralRatio(
            collateralAmount,
            debtAmount,
            price,
          ),
        };
```

Potentially null/undefined value for `collateralRatioSortKey_legacy` is what causes the typecheck error, this code resolves it - unless you were worried about empty values?